spec/valkyrie/resource_spec.rb

```diff
@@ -118,6 +118,7 @@ class MyLockingResource < Valkyrie::Resource
 
       it "has an optimistic_lock_token attribute" do
         expect(MyLockingResource.new).to respond_to(Valkyrie::Persistence::Attributes::OPTIMISTIC_LOCK)
+        expect(MyLockingResource.new.optimistic_locking_enabled?).to be_truthy
```

Comment:
Can these just be .to eq true and equivalent for false? I'd rather have a defined interface than rely on truthiness.